Examples/Modules/Chapter 12/Ex12_17/Truckload.cppm

```diff
@@ -0,0 +1,28 @@
+export module truckload;
+
+export import :shared_box;
+import :package;
```

Comment:
Shouldn't this be export import?
Ch11 says "the key is that every interface partition must eventually be exported from the module’s so-called primary module interface file" and truckload:package is an interface partition in package.cppm.

Reply:
It's not an interface partition, it's an implementation partition.

Reply:
as it is now, it's an interface partition right, it's in Package.cppm

Reply:
The file extension does not determine what type a file is, it is the way you compile it. Either way, .cpp or .cppm, the default compilation that VC++ chooses will be wrong. But I agree, it probably would make more sense to use .cpp for implementation/internal partitions.